setup.py

```diff
@@ -258,7 +274,7 @@ def get_export_symbols(self, ext):
     ext_modules = cythonize(
         [*cythonize_files], # This runs even if build_ext isn't invoked...
         nthreads=thread_count,
-        annotate=False,
+        annotate=True,
```

Comment:
Did you mean to commit this? Fine if so, just checking.

Reply:
You're right. That was a mistake